src/angular-app/bellows/shared/audio-recorder/audio-recorder.component.ts

```diff
@@ -1,163 +1,120 @@
+import { webmFixDuration } from 'webm-fix-duration';
 import * as angular from 'angular';
-import * as lamejs from 'lamejs';
-
-declare var webkitAudioContext: {
-  new(): AudioContext;
-};
 
 export class AudioRecorderController implements angular.IController {
 
   static $inject = ['$interval', '$scope'];
-  constructor(private $interval: angular.IIntervalService, private $scope: angular.IScope) {}
 
+  mediaRecorder: MediaRecorder;
+  chunks: any = [];
```

Comment:
I am uncertain if this is the right type, but it might work

```suggestion
  chunks: string[] = [];
```

Reply:
That works!